packages/loading-indicator/src/LoadingIndicator.story.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { css } from '@leafygreen-ui/emotion';
+import {
+  DarkModeProps,
+  storybookArgTypes,
+  StoryMeta,
+} from '@leafygreen-ui/lib';
+import { spacing } from '@leafygreen-ui/tokens';
+import { Body, InlineCode } from '@leafygreen-ui/typography';
+
+import BlobLoader from './BlobLoader';
+import { Spinner } from '.';
+
+export default StoryMeta({
+  title: 'Components/LoadingIndicator',
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    description: { control: 'text' },
+  },
+  parameters: {
+    default: 'LiveExample',
+    controls: {
+      exclude: ['className', 'baseFontSize'],
+    },
+  },
+});
+
+const storyRootStyles = css`
+  display: flex;
+  gap: 48px;
+  align-items: end;
+`;
+
+const variantContainerStyles = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const labelStyles = css`
+  margin-top: ${spacing[5]}px;
+`;
+
+const Template: ComponentStory<typeof Spinner> = (
+  props: {
+    description?: string;
+  } & DarkModeProps,
+) => (
+  <div className={storyRootStyles}>
```

Comment:
Why are we laying it out this way as opposed to setting `variant` in `args`?

Reply:
for the Live Example - this was requested by Rob